sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/ConditionEvaluatesToConstant.cs

```diff
@@ -128,7 +128,7 @@ public AnalysisContext(CSharpExplodedGraph explodedGraph, SyntaxNodeAnalysisCont
                 explodedGraph.ConditionEvaluated += ConditionEvaluatedHandler;
             }
 
-            public bool SupportPartialWalk { get; }
+            public bool SupportPartialResults { get; }
```

Comment:
sorry... it should be `SupportsPartialResults` (notice the s)

when asking the object `f.SupportsPartialResults`

`Support()` would be a command to the object (please support this)

and as a property "Support" is the plural form of the verb, we need the singular